src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -2509,7 +2509,7 @@ End Class
             Await VerifyItemExistsAsync(markup, "a")
             Await VerifyItemExistsAsync(markup, "b")
             Await VerifyItemExistsAsync(markup, "c")
-            Await VerifyItemIsAbsentAsync(markup, "Equals")
+            Await VerifyItemExistsAsync(markup, "Equals")
```

Comment:
This part isn't right - We shouldn't show static members from `System.Enum` on instances of actual Enum types.